cmd/image/qcow2ova/qcow2ova.go

```diff
@@ -39,6 +39,10 @@ Examples:
   pvsadm image qcow2ova --image-name centos-82 --image-dist centos --os-password s0meC0mplexPassword --image-url /root/CentOS-8-GenericCloud-8.2.2004-20200611.2.ppc64le.qcow2
 `,
 	PreRunE: func(cmd *cobra.Command, args []string) error {
+		if os.Geteuid() != 0 {
```

Comment:
use the validate framework as mentioned here - https://github.com/ppc64le-cloud/pvsadm/issues/40#issuecomment-740392004